awx/api/views/__init__.py

```diff
@@ -245,13 +245,6 @@ def get(self, request, format=None):
                                    'total': hg_projects.count(),
                                    'failed': hg_failed_projects.count()}
 
-        user_jobs = get_user_queryset(request.user, models.Job)
```

Comment:
@matburt @mabashian this is _really_ slow.  Do we need it?  What drives it (I removed it and the dashboard UI seemed tower just fine without it)?

Reply:
hmm, this should get the counts that the user is able to see... does this not drive the dashboard at all?

Reply:
I'm not really sure *what* this is used for:

![image](https://user-images.githubusercontent.com/214912/64655119-43d71f80-d3f9-11e9-8e0e-9fb058ae0972.png)

Reply:
The dash gets its graph counts from `DashboardJobsGraphView` and not `DashboardView`: https://github.com/ansible/awx/blob/devel/awx/ui/client/src/home/dashboard/graphs/job-status/job-status-graph.service.js#L23.

I don't see any place where we're using `data['jobs']` from /api/v2/dashboard` so I think dropping this query should be ok.

Reply:
![goforit](https://user-images.githubusercontent.com/9753817/64660973-b273a780-d410-11e9-8124-27dff4b8d8d1.gif)